tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/PackageRestore/PackageRestoreDataSourceTests.cs

```diff
@@ -94,7 +94,6 @@ private static PackageRestoreDataSource CreateInstance(UnconfiguredProject? proj
             solutionRestoreService ??= IVsSolutionRestoreServiceFactory.Create();
             IProjectDiagnosticOutputService logger = IProjectDiagnosticOutputServiceFactory.Create();
             IFileSystem fileSystem = IFileSystemFactory.Create();
-            var projectDependentFileChangeNotificationService = IProjectDependentFileChangeNotificationServiceFactory.Create();
```

Comment:
Does this leave any other usages of `IProjectDependentFileChangeNotificationServiceFactory`? If not, please remove it.